data/tmc-devices.json

```diff
@@ -195,19 +194,19 @@
                         },
```

Comment:
"simulationMode" is not included in attribute_properties of Subarray 2. The issue is not exactly at this line. Need to add above property in attribute_properties block of Subarray 2.

Reply:
'simulationMode' property added in attribute_properties of subarray2.